### ISSUE TITLE
Bees with custom reagents inside halt all work and retreat to the apiary forever.

### ISSUE THREAD

**Issue by Likteer** [tgstation/tgstation#73317]
## Round ID:
I forgot. Issue has been present for a while even in 2022.

## Reproduction:
Inject queen bee with 5u of smth before putting in the apiary.
The first bee pops up and begins harvesting resources from a nearby plant.
After gathering enough resources for a second bee, it suddently disappears.

Multiple tweaks to the hive (including removing the queen which should normally eject the bees) fail at fixing this.

Upon breaking the apiary, I found 2 regular bees and the queen. They stopped following their routine AI and would float in place. They became aggressive only after attacking the queen and waiting some more.

Default bees do not appear to suffer from this issue.

**Comment by timothymtorres** [comment]
Duplicate of #69315
